rasa/nlu/test.py

```diff
@@ -1022,12 +1023,17 @@ def get_entity_extractors(interpreter: Interpreter) -> Set[Text]:
 
     Processors are removed since they do not detect the boundaries themselves.
     """
-
     from rasa.nlu.extractors.extractor import EntityExtractor
 
-    extractors = {
-        c.name for c in interpreter.pipeline if isinstance(c, EntityExtractor)
-    }
+    extractors = set()
+    for c in interpreter.pipeline:
+        if isinstance(c, EntityExtractor):
+            if c.name == "DIETClassifier":
```

Comment:
isn't it better?
```suggestion
            if isinstance(c, DIETClassifier):
```

Reply:
shall we rather create `EntityExtractor.active` property? to make it DIET independent?

Reply:
Not sure if that is needed. Seems to be a bit too much as we just have DIET as a special case for now. Would reiterate on that once we have another edge case.